stablehlo/dialect/CMakeLists.txt

```diff
@@ -135,6 +135,8 @@ mlir_tablegen(StablehloEnums.h.inc -gen-enum-decls)
 mlir_tablegen(StablehloEnums.cpp.inc -gen-enum-defs)
 mlir_tablegen(StablehloAttrs.h.inc -gen-attrdef-decls)
 mlir_tablegen(StablehloAttrs.cpp.inc -gen-attrdef-defs)
+mlir_tablegen(StablehloTypeDefs.h.inc -gen-typedef-decls --typedefs-dialect=stablehlo)
```

Comment:
Looks like we're missing bazel build rules?

Reply:
🤦 Ah yep, thank you! Done. Sorry, I was only running the CMake build script, not the bazel build script.